apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx

```diff
@@ -89,6 +91,9 @@ export default async function Details({params}: Props) {
 
   const enrichments = await fetcher.getById(id);
   useObject.setState({objectId: object.id, locale, enrichments});
+  const enrichmentNames = enrichments?.filter(
```

Comment:
To be sure: what are 'enrichment names'? For example, are these enrichments about the name of an object? If so, rename to e.g. `enrichmentsAboutName` or `nameEnrichments`?